src/App.css

```diff
@@ -1,38 +1,168 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300&display=swap');
+
+* {
+  box-sizing: border-box;
+  font-family: 'Montserrat', 'Roboto Slab', sans-serif;
+  font-size: 16px;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #fafafa;
+  width: 100vw;
+  min-height: 100vh;
+  margin: 0 auto;
+  padding-top: 5vh;
+  overflow-x: hidden;
+}
+
 .App {
-  text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 28vw;
+  height: 70vh;
+  background-color: black;
+  border-radius: 30px;
+  padding: 3rem 1rem;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+/* ----------- Header ------------ */
+
+.header-container {
+  /* background-image: url("./feature/components/images/world.png"); */
```

Comment:
- [ ] I would recommend removing the commented line as after removing the codebase looks cleaner.

Reply:
> * [x]  I would recommend removing the commented line as after removing the codebase looks cleaner.

- [ ] OPTIONAL: This issue is not addressed. Kindly make the changes requested by the previous reviewer as unnecessarily commented lines of codes would make your code messy. I made it an optional change because I don't want to drag you back for such a silly mistake.